Detectors/ITSMFT/ITS/reconstruction/include/ITSReconstruction/CATracker.h

```diff
@@ -10,16 +10,18 @@
 #define ALIITSUCATRACKER_H
 
 #include <vector>
+#include <array>
 
 #include "ITSReconstruction/CAaux.h"
 #include "ITSReconstruction/CATrackingStation.h"
 #include "DetectorsBase/Track.h"
 
-using std::vector;
-
 namespace AliceO2 {
   namespace ITS {
     namespace CA {
+      using std::vector;
```

Comment:
I think this is not "forbidden" anywhere, as this does not mess with namespace resolution. It is equivalent to `using vector = std::vector;` - just an alias. Maybe there is a naming rule somewhere for type names, like `using Vector = std::vector` but in this case I would not bother, std:: is not too much to type, even in the interfaces. I would however strongly recommend using aliases (also in interfaces) for the more crazy typedefs (like boost::interprocess::offset_ptr<boost::interprocess::interprocess_condition> )...